autoflatten/viz.py

```diff
@@ -44,6 +48,129 @@ def compute_triangle_areas(vertices_2d, faces):
     return areas
 
 
+def compute_kring_distortion(
+    xy,
+    base_vertices,
+    base_faces,
+    orig_indices,
+    k=7,
+    n_samples_per_ring=12,
+    verbose=True,
+):
+    """Compute per-vertex metric distortion using k-ring geodesic distances.
+
+    Uses the same approach as autoflatten optimization:
+    - Computes geodesic distances to k-ring neighbors on 3D surface
+    - Compares with Euclidean distances on 2D flatmap
+    - Returns percentage error per vertex: 100 * mean(|d_2D - d_3D|) / mean(d_3D)
+
```

Comment:
Add: - This should match plots in Fischl et al., 1999, and the computation implemented in FreeSurfer